packages/packem/src/rollup/plugins/css/runtime/inject-css.ts

```
@@ -0,0 +1,84 @@
+    let styleTag;
+
+    if (singleTag) {
+        let id = containers.indexOf(container);
+
+        if (id === -1) {
+            id = containers.push(container) - 1;
+
+            styleTags[id] = {};
+        }
+
+        styleTag = styleTags[id]?.[position] ? styleTags[id][position] : (styleTags[id][position] = createStyleTag());
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)